gradle/dependencies.gradle

```diff
@@ -2,23 +2,23 @@ ext {
 
   androidVersions = [
       minSdkVersion    : 14,
-      targetSdkVersion : 27,
-      compileSdkVersion: 27,
+      targetSdkVersion : 28,
+      compileSdkVersion: 28,
       buildToolsVersion: '28.0.3'
```

Comment:
buils tools version can be dropped in total, this configuration has been made obsolete in latest Android gradle tool releases as they always match the gradle tooling with the build tools.